scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1229,8 +1336,8 @@
             ;; Straggler handling configured
             straggler-handling {:straggler-handling/type :straggler-handling.type/quantile-deviation
                                 :straggler-handling/parameters
-                                {:straggler-handling.quantile-deviation/quantile 0.5
-                                 :straggler-handling.quantile-deviation/multiplier 2.0}}
+                                                         {:straggler-handling.quantile-deviation/quantile 0.5
+                                                          :straggler-handling.quantile-deviation/multiplier 2.0}}
```

Comment:
Indentation seems off here

Reply:
autoformatting isn't changing anything, but I will put the map next to `:straggler-handling/paramaters`